src/mudlet.pro

```diff
@@ -128,6 +128,7 @@ win64 {
 # linux-g++-32, linux-g++-64
 
 
+TEMPLATE = app
```

Comment:
`app` is default already - no need to list it.

Reply:
It was already in the file I just relocated it...

Reply:
Ah, ok!